packages/web/src/components/admin_page/url_shortener/table.jsx

```diff
@@ -138,7 +138,7 @@ const UrlsTable = () => {
                     Create a shortened URL
                 </Button>
             </OverlayTrigger>
-            <Table style={{ color: 'white' }}>
+            <Table style={{ color: 'white', marginTop: '4vw' }}>
```

Comment:
Why is this needed?

Reply:
Otherwise the back button will appear on top of the text of the page. With the padding it moves it down a bit so that the button does not overlap with anything

Reply:
Isn't there a way to fix this within the button's styling?

Overlapping button is still present. 

<img width="817" alt="image" src="https://github.com/AUBGTheHUB/monolith/assets/104720011/36824104-2471-45da-bcde-68c76020feef">
<img width="554" alt="image" src="https://github.com/AUBGTheHUB/monolith/assets/104720011/c8edeb77-de91-477e-ab05-60cc8156f4f2">

Reply:
Now that you opened the mobile view. Do we need a back button for the mobile view? It seems a bit unnecessary and impractital.

Reply:
You have a point. Disable it on mobile resolutions.